lib/tasks/db.rake

```diff
@@ -68,7 +72,7 @@ ActiveRecord::MigrationProxy.prepend(ActualDbSchema::MigrationProxyPatch)
 namespace :db do
   desc "Rollback migrations that were run inside not a merged branch."
   task rollback_branches: :load_config do
-    ActiveRecord::Tasks::DatabaseTasks.raise_for_multi_db(command: "db:rollback_branches")
+    ActiveRecord::Tasks::DatabaseTasks.try(:raise_for_multi_db, command: "db:rollback_branches")
```

Comment:
Can it check for Rails version as well instead of using `try`? Looks like it was introduced in Rails 6 as well.

Reply:
done